reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
+  def print_multiple_times(msg, cantidad, cont, limit_inf, limit_sup) when cont <= limit_sup do
+    msg = msg <> "#{cont}"
```

Comment:
Pro tip, try to use, Elixir interpolation instead of variable string concatenation as a good practice 💡

Reply:
@kamala1606 after fixing it you can merge this PR :)